src/tests/topp/OpenSwathAssayGenerator_output.TraML

```diff
@@ -61,9 +61,6 @@
       <cvParam cvRef="MS" accession="MS:1001226" name="product ion intensity" value="10000"/>
       <cvParam cvRef="MS" accession="MS:1002007" name="target SRM transition"/>
       <userParam name="annotation" type="xsd:string" value="y7^2"/>
-      <userParam name="detecting_transition" type="xsd:string" value="true"/>
-      <userParam name="identifying_transition" type="xsd:integer" value="0"/>
-      <userParam name="quantifying_transition" type="xsd:integer" value="1"/>
```

Comment:
was this valid input? I now changed it everywhere to "xsd:boolean" and true/false